src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 100%;
+  flex-basis: auto;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav_list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  position: relative;
+  height: 60px;
+}
+
+.nav_list :not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_list a {
+  display: inline-block;
+  height: 60px;
+  vertical-align: middle;
+  text-align: center;
+  line-height: 60px;
+  color: black;
+  text-decoration: none;
+}
+
+.nav_list a:hover {
+  color: #00acdc;
+}
+
+.nav_list a:active {
+  color: #00acdc;
+}
+
+.nav_list .is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  display: inline-block;
```

Comment:
why do we need this?

Reply:
for position blue line, and in the conditions of the task  is indicated to use : after.